### ISSUE TITLE
Prefer https images over http

### ISSUE THREAD

**Issue by kspearrin** [mat/besticon#29]
https://icons.better-idea.org/icon?url=kayak.com&size=16..24..32

This returns http://a1.r9cdn.net/res/images/marble/apple-touch-icon-152x152.png?v=3ac057ec0063bda95b6097a49774dd6d5a11fb6a

It would be nice if we could get the https version instead:

https://a1.r9cdn.net/res/images/marble/apple-touch-icon-152x152.png?v=3ac057ec0063bda95b6097a49774dd6d5a11fb6a

**Comment by mat** [comment]
Yeah, good suggestion. I've been thinking of this ever since I started serving icons.better-idea.org over https.

Unfortunately it does not seem to help to add an explicit scheme in your example:
https://icons.better-idea.org/icon?url=https://kayak.com&size=16..24..32

Which looks like for the best result I'd have to just try https first http second, which is not really optimal.
